src/common/ManagedCommon/ManagedCommon.csproj

```diff
@@ -14,10 +14,17 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
-    <PackageReference Include="System.Management" Version="6.0.0" />
+    <PackageReference Include="System.Management" Version="7.0.0-rc.1.22426.10" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\ManagedTelemetry\Telemetry\ManagedTelemetry.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0-preview1.22403.2">

```

Comment:
Analyzers are now centralized
https://github.com/microsoft/PowerToys/blob/6d531a9a6bf2a5861e9fa338d2e68e18e318c1f8/Directory.Build.props#L39

Reply:
Thanks for that find @davidegiacometti !